Gems/EditorPythonBindings/Code/Source/PythonUtility.cpp

```diff
@@ -156,6 +156,10 @@ namespace EditorPythonBindings
                 AZ::s64 outboundPythonValue = 0;
 
                 bool converted = 
+                    ConvertPythonFromEnumClass<short>(underlyingTypeId, behaviorValue, outboundPythonValue) ||
+                    ConvertPythonFromEnumClass<unsigned short>(underlyingTypeId, behaviorValue, outboundPythonValue) ||
```

Comment:
The short` and `unsigned short` type is covered by the `AZ::s16` and `AZ::u16` type aliases as they are always the `short` type.